// rdf-namespaces/dist/as-3793f0cb.js
/**
 * Accept
 *
 * Actor accepts the Object
 *
 * http://www.w3.org/ns/activitystreams#Accept
 */
var Accept = 'http://www.w3.org/ns/activitystreams#Accept';
/**
 * Activity
 *
 * An Object representing some form of Action that has been taken
 *
 * http://www.w3.org/ns/activitystreams#Activity
 */
var Activity = 'http://www.w3.org/ns/activitystreams#Activity';
/**
 * Block
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#Block
 */
var Block = 'http://www.w3.org/ns/activitystreams#Block';
/**
 * IntransitiveActivity
 *
 * An Activity that has no direct object
 *
 * http://www.w3.org/ns/activitystreams#IntransitiveActivity
 */
var IntransitiveActivity = 'http://www.w3.org/ns/activitystreams#IntransitiveActivity';
/**
 * Add
 *
 * To Add an Object or Link to Something
 *
 * http://www.w3.org/ns/activitystreams#Add
 */
var Add = 'http://www.w3.org/ns/activitystreams#Add';
/**
 * Announce
 *
 * Actor announces the object to the target
 *
 * http://www.w3.org/ns/activitystreams#Announce
 */
var Announce = 'http://www.w3.org/ns/activitystreams#Announce';
/**
 * Application
 *
 * Represents a software application of any sort
 *
 * http://www.w3.org/ns/activitystreams#Application
 */
var Application = 'http://www.w3.org/ns/activitystreams#Application';
/**
 * Arrive
 *
 * To Arrive Somewhere (can be used, for instance, to indicate that a particular entity is currently located somewhere, e.g. a "check-in")
 *
 * http://www.w3.org/ns/activitystreams#Arrive
 */
var Arrive = 'http://www.w3.org/ns/activitystreams#Arrive';
/**
 * Article
 *
 * A written work. Typically several paragraphs long. For example, a blog post or a news article.
 *
 * http://www.w3.org/ns/activitystreams#Article
 */
var Article = 'http://www.w3.org/ns/activitystreams#Article';
/**
 * Audio
 *
 * An audio file
 *
 * http://www.w3.org/ns/activitystreams#Audio
 */
var Audio = 'http://www.w3.org/ns/activitystreams#Audio';
/**
 * Collection
 *
 * An ordered or unordered collection of Objects or Links
 *
 * http://www.w3.org/ns/activitystreams#Collection
 */
var Collection = 'http://www.w3.org/ns/activitystreams#Collection';
/**
 * CollectionPage
 *
 * A subset of items from a Collection
 *
 * http://www.w3.org/ns/activitystreams#CollectionPage
 */
var CollectionPage = 'http://www.w3.org/ns/activitystreams#CollectionPage';
/**
 * OrderedCollectionPage
 *
 * An ordered subset of items from an OrderedCollection
 *
 * http://www.w3.org/ns/activitystreams#OrderedCollectionPage
 */
var OrderedCollectionPage = 'http://www.w3.org/ns/activitystreams#OrderedCollectionPage';
/**
 * Relationship
 *
 * Represents a Social Graph relationship between two Individuals (indicated by the 'a' and 'b' properties)
 *
 * http://www.w3.org/ns/activitystreams#Relationship
 */
var Relationship = 'http://www.w3.org/ns/activitystreams#Relationship';
/**
 * Create
 *
 * To Create Something
 *
 * http://www.w3.org/ns/activitystreams#Create
 */
var Create = 'http://www.w3.org/ns/activitystreams#Create';
/**
 * Delete
 *
 * To Delete Something
 *
 * http://www.w3.org/ns/activitystreams#Delete
 */
var Delete = 'http://www.w3.org/ns/activitystreams#Delete';
/**
 * Dislike
 *
 * The actor dislikes the object
 *
 * http://www.w3.org/ns/activitystreams#Dislike
 */
var Dislike = 'http://www.w3.org/ns/activitystreams#Dislike';
/**
 * Document
 *
 * Represents a digital document/file of any sort
 *
 * http://www.w3.org/ns/activitystreams#Document
 */
var Document = 'http://www.w3.org/ns/activitystreams#Document';
/**
 * Event
 *
 * An Event of any kind
 *
 * http://www.w3.org/ns/activitystreams#Event
 */
var Event = 'http://www.w3.org/ns/activitystreams#Event';
/**
 * Flag
 *
 * To flag something (e.g. flag as inappropriate, flag as spam, etc)
 *
 * http://www.w3.org/ns/activitystreams#Flag
 */
var Flag = 'http://www.w3.org/ns/activitystreams#Flag';
/**
 * Follow
 *
 * To Express Interest in Something
 *
 * http://www.w3.org/ns/activitystreams#Follow
 */
var Follow = 'http://www.w3.org/ns/activitystreams#Follow';
/**
 * Group
 *
 * A Group of any kind.
 *
 * http://www.w3.org/ns/activitystreams#Group
 */
var Group = 'http://www.w3.org/ns/activitystreams#Group';
/**
 * Ignore
 *
 * Actor is ignoring the Object
 *
 * http://www.w3.org/ns/activitystreams#Ignore
 */
var Ignore = 'http://www.w3.org/ns/activitystreams#Ignore';
/**
 * Image
 *
 * An Image file
 *
 * http://www.w3.org/ns/activitystreams#Image
 */
var Image = 'http://www.w3.org/ns/activitystreams#Image';
/**
 * Invite
 *
 * To invite someone or something to something
 *
 * http://www.w3.org/ns/activitystreams#Invite
 */
var Invite = 'http://www.w3.org/ns/activitystreams#Invite';
/**
 * Join
 *
 * To Join Something
 *
 * http://www.w3.org/ns/activitystreams#Join
 */
var Join = 'http://www.w3.org/ns/activitystreams#Join';
/**
 * Leave
 *
 * To Leave Something
 *
 * http://www.w3.org/ns/activitystreams#Leave
 */
var Leave = 'http://www.w3.org/ns/activitystreams#Leave';
/**
 * Like
 *
 * To Like Something
 *
 * http://www.w3.org/ns/activitystreams#Like
 */
var Like = 'http://www.w3.org/ns/activitystreams#Like';
/**
 * View
 *
 * The actor viewed the object
 *
 * http://www.w3.org/ns/activitystreams#View
 */
var View = 'http://www.w3.org/ns/activitystreams#View';
/**
 * Listen
 *
 * The actor listened to the object
 *
 * http://www.w3.org/ns/activitystreams#Listen
 */
var Listen = 'http://www.w3.org/ns/activitystreams#Listen';
/**
 * Read
 *
 * The actor read the object
 *
 * http://www.w3.org/ns/activitystreams#Read
 */
var Read = 'http://www.w3.org/ns/activitystreams#Read';
/**
 * Move
 *
 * The actor is moving the object. The target specifies where the object is moving to. The origin specifies where the object is moving from.
 *
 * http://www.w3.org/ns/activitystreams#Move
 */
var Move = 'http://www.w3.org/ns/activitystreams#Move';
/**
 * Travel
 *
 * The actor is traveling to the target. The origin specifies where the actor is traveling from.
 *
 * http://www.w3.org/ns/activitystreams#Travel
 */
var Travel = 'http://www.w3.org/ns/activitystreams#Travel';
/**
 * Link
 *
 * Represents a qualified reference to another resource. Patterned after the RFC5988 Web Linking Model
 *
 * http://www.w3.org/ns/activitystreams#Link
 */
var Link = 'http://www.w3.org/ns/activitystreams#Link';
/**
 * Mention
 *
 * A specialized Link that represents an @mention
 *
 * http://www.w3.org/ns/activitystreams#Mention
 */
var Mention = 'http://www.w3.org/ns/activitystreams#Mention';
/**
 * Note
 *
 * A Short note, typically less than a single paragraph. A "tweet" is an example, or a "status update"
 *
 * http://www.w3.org/ns/activitystreams#Note
 */
var Note = 'http://www.w3.org/ns/activitystreams#Note';
/**
 * Object
 *
 * null
 * Note that `Object` is a reserved Javascript keyword, and is therefore suffixed by `__workaround`.
 * For a list of reserved keywords, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords.
 *
 * http://www.w3.org/ns/activitystreams#Object
 */
var Object__workaround = 'http://www.w3.org/ns/activitystreams#Object';
/**
 * Offer
 *
 * To Offer something to someone or something
 *
 * http://www.w3.org/ns/activitystreams#Offer
 */
var Offer = 'http://www.w3.org/ns/activitystreams#Offer';
/**
 * OrderedCollection
 *
 * A variation of Collection in which items are strictly ordered
 *
 * http://www.w3.org/ns/activitystreams#OrderedCollection
 */
var OrderedCollection = 'http://www.w3.org/ns/activitystreams#OrderedCollection';
/**
 * OrderedItems
 *
 * A rdf:List variant for Objects and Links
 *
 * http://www.w3.org/ns/activitystreams#OrderedItems
 */
var OrderedItems = 'http://www.w3.org/ns/activitystreams#OrderedItems';
/**
 * Page
 *
 * A Web Page
 *
 * http://www.w3.org/ns/activitystreams#Page
 */
var Page = 'http://www.w3.org/ns/activitystreams#Page';
/**
 * Person
 *
 * A Person
 *
 * http://www.w3.org/ns/activitystreams#Person
 */
var Person = 'http://www.w3.org/ns/activitystreams#Person';
/**
 * Organization
 *
 * An Organization
 *
 * http://www.w3.org/ns/activitystreams#Organization
 */
var Organization = 'http://www.w3.org/ns/activitystreams#Organization';
/**
 * Profile
 *
 * A Profile Document
 *
 * http://www.w3.org/ns/activitystreams#Profile
 */
var Profile = 'http://www.w3.org/ns/activitystreams#Profile';
/**
 * Place
 *
 * A physical or logical location
 *
 * http://www.w3.org/ns/activitystreams#Place
 */
var Place = 'http://www.w3.org/ns/activitystreams#Place';
/**
 * Question
 *
 * A question of any sort.
 *
 * http://www.w3.org/ns/activitystreams#Question
 */
var Question = 'http://www.w3.org/ns/activitystreams#Question';
/**
 * Reject
 *
 * Actor rejects the Object
 *
 * http://www.w3.org/ns/activitystreams#Reject
 */
var Reject = 'http://www.w3.org/ns/activitystreams#Reject';
/**
 * Remove
 *
 * To Remove Something
 *
 * http://www.w3.org/ns/activitystreams#Remove
 */
var Remove = 'http://www.w3.org/ns/activitystreams#Remove';
/**
 * Service
 *
 * A service provided by some entity
 *
 * http://www.w3.org/ns/activitystreams#Service
 */
var Service = 'http://www.w3.org/ns/activitystreams#Service';
/**
 * TentativeAccept
 *
 * Actor tentatively accepts the Object
 *
 * http://www.w3.org/ns/activitystreams#TentativeAccept
 */
var TentativeAccept = 'http://www.w3.org/ns/activitystreams#TentativeAccept';
/**
 * TentativeReject
 *
 * Actor tentatively rejects the object
 *
 * http://www.w3.org/ns/activitystreams#TentativeReject
 */
var TentativeReject = 'http://www.w3.org/ns/activitystreams#TentativeReject';
/**
 * Tombstone
 *
 * A placeholder for a deleted object
 *
 * http://www.w3.org/ns/activitystreams#Tombstone
 */
var Tombstone = 'http://www.w3.org/ns/activitystreams#Tombstone';
/**
 * Undo
 *
 * To Undo Something. This would typically be used to indicate that a previous Activity has been undone.
 *
 * http://www.w3.org/ns/activitystreams#Undo
 */
var Undo = 'http://www.w3.org/ns/activitystreams#Undo';
/**
 * Update
 *
 * To Update/Modify Something
 *
 * http://www.w3.org/ns/activitystreams#Update
 */
var Update = 'http://www.w3.org/ns/activitystreams#Update';
/**
 * Video
 *
 * A Video document of any kind.
 *
 * http://www.w3.org/ns/activitystreams#Video
 */
var Video = 'http://www.w3.org/ns/activitystreams#Video';
/**
 * actor
 *
 * Subproperty of as:attributedTo that identifies the primary actor
 *
 * http://www.w3.org/ns/activitystreams#actor
 */
var actor = 'http://www.w3.org/ns/activitystreams#actor';
/**
 * attributedTo
 *
 * Identifies an entity to which an object is attributed
 *
 * http://www.w3.org/ns/activitystreams#attributedTo
 */
var attributedTo = 'http://www.w3.org/ns/activitystreams#attributedTo';
/**
 * attachment
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#attachment
 */
var attachment = 'http://www.w3.org/ns/activitystreams#attachment';
/**
 * attachments
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#attachments
 */
var attachments = 'http://www.w3.org/ns/activitystreams#attachments';
/**
 * author
 *
 * Identifies the author of an object. Deprecated. Use as:attributedTo instead
 *
 * http://www.w3.org/ns/activitystreams#author
 */
var author = 'http://www.w3.org/ns/activitystreams#author';
/**
 * bcc
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#bcc
 */
var bcc = 'http://www.w3.org/ns/activitystreams#bcc';
/**
 * bto
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#bto
 */
var bto = 'http://www.w3.org/ns/activitystreams#bto';
/**
 * cc
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#cc
 */
var cc = 'http://www.w3.org/ns/activitystreams#cc';
/**
 * context
 *
 * Specifies the context within which an object exists or an activity was performed
 *
 * http://www.w3.org/ns/activitystreams#context
 */
var context = 'http://www.w3.org/ns/activitystreams#context';
/**
 * current
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#current
 */
var current = 'http://www.w3.org/ns/activitystreams#current';
/**
 * first
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#first
 */
var first = 'http://www.w3.org/ns/activitystreams#first';
/**
 * generator
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#generator
 */
var generator = 'http://www.w3.org/ns/activitystreams#generator';
/**
 * icon
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#icon
 */
var icon = 'http://www.w3.org/ns/activitystreams#icon';
/**
 * image
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#image
 */
var image = 'http://www.w3.org/ns/activitystreams#image';
/**
 * inReplyTo
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#inReplyTo
 */
var inReplyTo = 'http://www.w3.org/ns/activitystreams#inReplyTo';
/**
 * items
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#items
 */
var items = 'http://www.w3.org/ns/activitystreams#items';
/**
 * last
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#last
 */
var last = 'http://www.w3.org/ns/activitystreams#last';
/**
 * location
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#location
 */
var location = 'http://www.w3.org/ns/activitystreams#location';
/**
 * next
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#next
 */
var next = 'http://www.w3.org/ns/activitystreams#next';
/**
 * object
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#object
 */
var object = 'http://www.w3.org/ns/activitystreams#object';
/**
 * oneOf
 *
 * Describes a possible exclusive answer or option for a question.
 *
 * http://www.w3.org/ns/activitystreams#oneOf
 */
var oneOf = 'http://www.w3.org/ns/activitystreams#oneOf';
/**
 * oneOf
 *
 * Describes a possible inclusive answer or option for a question.
 *
 * http://www.w3.org/ns/activitystreams#anyOf
 */
var anyOf = 'http://www.w3.org/ns/activitystreams#anyOf';
/**
 * prev
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#prev
 */
var prev = 'http://www.w3.org/ns/activitystreams#prev';
/**
 * preview
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#preview
 */
var preview = 'http://www.w3.org/ns/activitystreams#preview';
/**
 * provider
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#provider
 */
var provider = 'http://www.w3.org/ns/activitystreams#provider';
/**
 * replies
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#replies
 */
var replies = 'http://www.w3.org/ns/activitystreams#replies';
/**
 * result
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#result
 */
var result = 'http://www.w3.org/ns/activitystreams#result';
/**
 * audience
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#audience
 */
var audience = 'http://www.w3.org/ns/activitystreams#audience';
/**
 * partOf
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#partOf
 */
var partOf = 'http://www.w3.org/ns/activitystreams#partOf';
/**
 * tag
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#tag
 */
var tag = 'http://www.w3.org/ns/activitystreams#tag';
/**
 * tags
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#tags
 */
var tags = 'http://www.w3.org/ns/activitystreams#tags';
/**
 * target
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#target
 */
var target = 'http://www.w3.org/ns/activitystreams#target';
/**
 * origin
 *
 * For certain activities, specifies the entity from which the action is directed.
 *
 * http://www.w3.org/ns/activitystreams#origin
 */
var origin = 'http://www.w3.org/ns/activitystreams#origin';
/**
 * instrument
 *
 * Indentifies an object used (or to be used) to complete an activity
 *
 * http://www.w3.org/ns/activitystreams#instrument
 */
var instrument = 'http://www.w3.org/ns/activitystreams#instrument';
/**
 * to
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#to
 */
var to = 'http://www.w3.org/ns/activitystreams#to';
/**
 * url
 *
 * Specifies a link to a specific representation of the Object
 *
 * http://www.w3.org/ns/activitystreams#url
 */
var url = 'http://www.w3.org/ns/activitystreams#url';
/**
 * a
 *
 * On a Relationship object, identifies the subject. e.g. when saying "John is connected to Sally", 'subject' refers to 'John'
 *
 * http://www.w3.org/ns/activitystreams#subject
 */
var subject = 'http://www.w3.org/ns/activitystreams#subject';
/**
 * relationship
 *
 * On a Relationship object, describes the type of relationship
 *
 * http://www.w3.org/ns/activitystreams#relationship
 */
var relationship = 'http://www.w3.org/ns/activitystreams#relationship';
/**
 * describes
 *
 * On a Profile object, describes the object described by the profile
 *
 * http://www.w3.org/ns/activitystreams#describes
 */
var describes = 'http://www.w3.org/ns/activitystreams#describes';
/**
 * formerType
 *
 * On a Tombstone object, describes the former type of the deleted object
 *
 * http://www.w3.org/ns/activitystreams#formerType
 */
var formerType = 'http://www.w3.org/ns/activitystreams#formerType';
/**
 * accuracy
 *
 * Specifies the accuracy around the point established by the longitude and latitude
 *
 * http://www.w3.org/ns/activitystreams#accuracy
 */
var accuracy = 'http://www.w3.org/ns/activitystreams#accuracy';
/**
 * altitude
 *
 * The altitude of a place
 *
 * http://www.w3.org/ns/activitystreams#altitude
 */
var altitude = 'http://www.w3.org/ns/activitystreams#altitude';
/**
 * content
 *
 * The content of the object.
 *
 * http://www.w3.org/ns/activitystreams#content
 */
var content = 'http://www.w3.org/ns/activitystreams#content';
/**
 * name
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#name
 */
var name = 'http://www.w3.org/ns/activitystreams#name';
/**
 * downstreamDuplicates
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#downstreamDuplicates
 */
var downstreamDuplicates = 'http://www.w3.org/ns/activitystreams#downstreamDuplicates';
/**
 * duration
 *
 * The duration of the object
 *
 * http://www.w3.org/ns/activitystreams#duration
 */
var duration = 'http://www.w3.org/ns/activitystreams#duration';
/**
 * endTime
 *
 * The ending time of the object
 *
 * http://www.w3.org/ns/activitystreams#endTime
 */
var endTime = 'http://www.w3.org/ns/activitystreams#endTime';
/**
 * height
 *
 * The display height expressed as device independent pixels
 *
 * http://www.w3.org/ns/activitystreams#height
 */
var height = 'http://www.w3.org/ns/activitystreams#height';
/**
 * href
 *
 * The target URI of the Link
 *
 * http://www.w3.org/ns/activitystreams#href
 */
var href = 'http://www.w3.org/ns/activitystreams#href';
/**
 * hreflang
 *
 * A hint about the language of the referenced resource
 *
 * http://www.w3.org/ns/activitystreams#hreflang
 */
var hreflang = 'http://www.w3.org/ns/activitystreams#hreflang';
/**
 * id
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#id
 */
var id = 'http://www.w3.org/ns/activitystreams#id';
/**
 * latitude
 *
 * The latitude
 *
 * http://www.w3.org/ns/activitystreams#latitude
 */
var latitude = 'http://www.w3.org/ns/activitystreams#latitude';
/**
 * longitude
 *
 * The longitude
 *
 * http://www.w3.org/ns/activitystreams#longitude
 */
var longitude = 'http://www.w3.org/ns/activitystreams#longitude';
/**
 * mediaType
 *
 * The MIME Media Type
 *
 * http://www.w3.org/ns/activitystreams#mediaType
 */
var mediaType = 'http://www.w3.org/ns/activitystreams#mediaType';
/**
 * objectType
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#objectType
 */
var objectType = 'http://www.w3.org/ns/activitystreams#objectType';
/**
 * published
 *
 * Specifies the date and time the object was published
 *
 * http://www.w3.org/ns/activitystreams#published
 */
var published = 'http://www.w3.org/ns/activitystreams#published';
/**
 * radius
 *
 * Specifies a radius around the point established by the longitude and latitude
 *
 * http://www.w3.org/ns/activitystreams#radius
 */
var radius = 'http://www.w3.org/ns/activitystreams#radius';
/**
 * rating
 *
 * A numeric rating (>= 0.0, <= 5.0) for the object
 *
 * http://www.w3.org/ns/activitystreams#rating
 */
var rating = 'http://www.w3.org/ns/activitystreams#rating';
/**
 * rel
 *
 * The RFC 5988 or HTML5 Link Relation associated with the Link
 *
 * http://www.w3.org/ns/activitystreams#rel
 */
var rel = 'http://www.w3.org/ns/activitystreams#rel';
/**
 * startIndex
 *
 * In a strictly ordered logical collection, specifies the index position of the first item in the items list
 *
 * http://www.w3.org/ns/activitystreams#startIndex
 */
var startIndex = 'http://www.w3.org/ns/activitystreams#startIndex';
/**
 * startTime
 *
 * The starting time of the object
 *
 * http://www.w3.org/ns/activitystreams#startTime
 */
var startTime = 'http://www.w3.org/ns/activitystreams#startTime';
/**
 * summary
 *
 * A short summary of the object
 *
 * http://www.w3.org/ns/activitystreams#summary
 */
var summary = 'http://www.w3.org/ns/activitystreams#summary';
/**
 * totalItems
 *
 * The total number of items in a logical collection
 *
 * http://www.w3.org/ns/activitystreams#totalItems
 */
var totalItems = 'http://www.w3.org/ns/activitystreams#totalItems';
/**
 * units
 *
 * Identifies the unit of measurement used by the radius, altitude and accuracy properties. The value can be expressed either as one of a set of predefined units or as a well-known common URI that identifies units.
 *
 * http://www.w3.org/ns/activitystreams#units
 */
var units = 'http://www.w3.org/ns/activitystreams#units';
/**
 * updated
 *
 * Specifies when the object was last updated
 *
 * http://www.w3.org/ns/activitystreams#updated
 */
var updated = 'http://www.w3.org/ns/activitystreams#updated';
/**
 * upstreamDuplicates
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#upstreamDuplicates
 */
var upstreamDuplicates = 'http://www.w3.org/ns/activitystreams#upstreamDuplicates';
/**
 * verb
 *
 *
 *
 * http://www.w3.org/ns/activitystreams#verb
 */
var verb = 'http://www.w3.org/ns/activitystreams#verb';
/**
 * width
 *
 * Specifies the preferred display width of the content, expressed in terms of device independent pixels.
 *
 * http://www.w3.org/ns/activitystreams#width
 */
var width = 'http://www.w3.org/ns/activitystreams#width';
/**
 * deleted
 *
 * Specifies the date and time the object was deleted
 *
 * http://www.w3.org/ns/activitystreams#deleted
 */
var deleted = 'http://www.w3.org/ns/activitystreams#deleted';

var asImport = /*#__PURE__*/Object.freeze({
	Accept: Accept,
	Activity: Activity,
	Block: Block,
	IntransitiveActivity: IntransitiveActivity,
	Add: Add,
	Announce: Announce,
	Application: Application,
	Arrive: Arrive,
	Article: Article,
	Audio: Audio,
	Collection: Collection,
	CollectionPage: CollectionPage,
	OrderedCollectionPage: OrderedCollectionPage,
	Relationship: Relationship,
	Create: Create,
	Delete: Delete,
	Dislike: Dislike,
	Document: Document,
	Event: Event,
	Flag: Flag,
	Follow: Follow,
	Group: Group,
	Ignore: Ignore,
	Image: Image,
	Invite: Invite,
	Join: Join,
	Leave: Leave,
	Like: Like,
	View: View,
	Listen: Listen,
	Read: Read,
	Move: Move,
	Travel: Travel,
	Link: Link,
	Mention: Mention,
	Note: Note,
	Object__workaround: Object__workaround,
	Offer: Offer,
	OrderedCollection: OrderedCollection,
	OrderedItems: OrderedItems,
	Page: Page,
	Person: Person,
	Organization: Organization,
	Profile: Profile,
	Place: Place,
	Question: Question,
	Reject: Reject,
	Remove: Remove,
	Service: Service,
	TentativeAccept: TentativeAccept,
	TentativeReject: TentativeReject,
	Tombstone: Tombstone,
	Undo: Undo,
	Update: Update,
	Video: Video,
	actor: actor,
	attributedTo: attributedTo,
	attachment: attachment,
	attachments: attachments,
	author: author,
	bcc: bcc,
	bto: bto,
	cc: cc,
	context: context,
	current: current,
	first: first,
	generator: generator,
	icon: icon,
	image: image,
	inReplyTo: inReplyTo,
	items: items,
	last: last,
	location: location,
	next: next,
	object: object,
	oneOf: oneOf,
	anyOf: anyOf,
	prev: prev,
	preview: preview,
	provider: provider,
	replies: replies,
	result: result,
	audience: audience,
	partOf: partOf,
	tag: tag,
	tags: tags,
	target: target,
	origin: origin,
	instrument: instrument,
	to: to,
	url: url,
	subject: subject,
	relationship: relationship,
	describes: describes,
	formerType: formerType,
	accuracy: accuracy,
	altitude: altitude,
	content: content,
	name: name,
	downstreamDuplicates: downstreamDuplicates,
	duration: duration,
	endTime: endTime,
	height: height,
	href: href,
	hreflang: hreflang,
	id: id,
	latitude: latitude,
	longitude: longitude,
	mediaType: mediaType,
	objectType: objectType,
	published: published,
	radius: radius,
	rating: rating,
	rel: rel,
	startIndex: startIndex,
	startTime: startTime,
	summary: summary,
	totalItems: totalItems,
	units: units,
	updated: updated,
	upstreamDuplicates: upstreamDuplicates,
	verb: verb,
	width: width,
	deleted: deleted
});

export { Update as $, Accept as A, Block as B, Collection as C, Delete as D, Event as E, Flag as F, Group as G, Profile as H, IntransitiveActivity as I, Join as J, Place as K, Leave as L, Move as M, Note as N, OrderedCollectionPage as O, Page as P, Question as Q, Relationship as R, Reject as S, Travel as T, Remove as U, View as V, Service as W, TentativeAccept as X, TentativeReject as Y, Tombstone as Z, Undo as _, Activity as a, totalItems as a$, Video as a0, actor as a1, attributedTo as a2, attachment as a3, attachments as a4, author as a5, bcc as a6, bto as a7, cc as a8, context as a9, url as aA, subject as aB, relationship as aC, describes as aD, formerType as aE, accuracy as aF, altitude as aG, content as aH, name as aI, downstreamDuplicates as aJ, duration as aK, endTime as aL, height as aM, href as aN, hreflang as aO, id as aP, latitude as aQ, longitude as aR, mediaType as aS, objectType as aT, published as aU, radius as aV, rating as aW, rel as aX, startIndex as aY, startTime as aZ, summary as a_, current as aa, first as ab, generator as ac, icon as ad, image as ae, inReplyTo as af, items as ag, last as ah, location as ai, next as aj, object as ak, oneOf as al, anyOf as am, prev as an, preview as ao, provider as ap, replies as aq, result as ar, audience as as, partOf as at, tag as au, tags as av, target as aw, origin as ax, instrument as ay, to as az, Add as b, units as b0, updated as b1, upstreamDuplicates as b2, verb as b3, width as b4, deleted as b5, asImport as b6, Announce as c, Application as d, Arrive as e, Article as f, Audio as g, CollectionPage as h, Create as i, Dislike as j, Document as k, Follow as l, Ignore as m, Image as n, Invite as o, Like as p, Listen as q, Read as r, Link as s, Mention as t, Object__workaround as u, Offer as v, OrderedCollection as w, OrderedItems as x, Person as y, Organization as z };
