// rdf-namespaces/dist/as.ts
type Property = string;
type Class = string;
type Datatype = string;
type OwlClass = string;
type OwlObjectProperty = string;
type OwlDatatypeProperty = string;
type HydraResource = string;
type HydraClass = string;
type HydraLink = string;
type HydraTemplatedLink = string;
type HydraVariableRepresentation = string;

/**
 * Accept
 * 
 * Actor accepts the Object
 *
 * http://www.w3.org/ns/activitystreams#Accept
 */
export const Accept: OwlClass = 'http://www.w3.org/ns/activitystreams#Accept';

/**
 * Activity
 * 
 * An Object representing some form of Action that has been taken
 *
 * http://www.w3.org/ns/activitystreams#Activity
 */
export const Activity: OwlClass = 'http://www.w3.org/ns/activitystreams#Activity';

/**
 * Block
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#Block
 */
export const Block: OwlClass = 'http://www.w3.org/ns/activitystreams#Block';

/**
 * IntransitiveActivity
 * 
 * An Activity that has no direct object
 *
 * http://www.w3.org/ns/activitystreams#IntransitiveActivity
 */
export const IntransitiveActivity: OwlClass = 'http://www.w3.org/ns/activitystreams#IntransitiveActivity';

/**
 * Add
 * 
 * To Add an Object or Link to Something
 *
 * http://www.w3.org/ns/activitystreams#Add
 */
export const Add: OwlClass = 'http://www.w3.org/ns/activitystreams#Add';

/**
 * Announce
 * 
 * Actor announces the object to the target
 *
 * http://www.w3.org/ns/activitystreams#Announce
 */
export const Announce: OwlClass = 'http://www.w3.org/ns/activitystreams#Announce';

/**
 * Application
 * 
 * Represents a software application of any sort
 *
 * http://www.w3.org/ns/activitystreams#Application
 */
export const Application: OwlClass = 'http://www.w3.org/ns/activitystreams#Application';

/**
 * Arrive
 * 
 * To Arrive Somewhere (can be used, for instance, to indicate that a particular entity is currently located somewhere, e.g. a "check-in")
 *
 * http://www.w3.org/ns/activitystreams#Arrive
 */
export const Arrive: OwlClass = 'http://www.w3.org/ns/activitystreams#Arrive';

/**
 * Article
 * 
 * A written work. Typically several paragraphs long. For example, a blog post or a news article.
 *
 * http://www.w3.org/ns/activitystreams#Article
 */
export const Article: OwlClass = 'http://www.w3.org/ns/activitystreams#Article';

/**
 * Audio
 * 
 * An audio file
 *
 * http://www.w3.org/ns/activitystreams#Audio
 */
export const Audio: OwlClass = 'http://www.w3.org/ns/activitystreams#Audio';

/**
 * Collection
 * 
 * An ordered or unordered collection of Objects or Links
 *
 * http://www.w3.org/ns/activitystreams#Collection
 */
export const Collection: OwlClass = 'http://www.w3.org/ns/activitystreams#Collection';

/**
 * CollectionPage
 * 
 * A subset of items from a Collection
 *
 * http://www.w3.org/ns/activitystreams#CollectionPage
 */
export const CollectionPage: OwlClass = 'http://www.w3.org/ns/activitystreams#CollectionPage';

/**
 * OrderedCollectionPage
 * 
 * An ordered subset of items from an OrderedCollection
 *
 * http://www.w3.org/ns/activitystreams#OrderedCollectionPage
 */
export const OrderedCollectionPage: OwlClass = 'http://www.w3.org/ns/activitystreams#OrderedCollectionPage';

/**
 * Relationship
 * 
 * Represents a Social Graph relationship between two Individuals (indicated by the 'a' and 'b' properties)
 *
 * http://www.w3.org/ns/activitystreams#Relationship
 */
export const Relationship: OwlClass = 'http://www.w3.org/ns/activitystreams#Relationship';

/**
 * Create
 * 
 * To Create Something
 *
 * http://www.w3.org/ns/activitystreams#Create
 */
export const Create: OwlClass = 'http://www.w3.org/ns/activitystreams#Create';

/**
 * Delete
 * 
 * To Delete Something
 *
 * http://www.w3.org/ns/activitystreams#Delete
 */
export const Delete: OwlClass = 'http://www.w3.org/ns/activitystreams#Delete';

/**
 * Dislike
 * 
 * The actor dislikes the object
 *
 * http://www.w3.org/ns/activitystreams#Dislike
 */
export const Dislike: OwlClass = 'http://www.w3.org/ns/activitystreams#Dislike';

/**
 * Document
 * 
 * Represents a digital document/file of any sort
 *
 * http://www.w3.org/ns/activitystreams#Document
 */
export const Document: OwlClass = 'http://www.w3.org/ns/activitystreams#Document';

/**
 * Event
 * 
 * An Event of any kind
 *
 * http://www.w3.org/ns/activitystreams#Event
 */
export const Event: OwlClass = 'http://www.w3.org/ns/activitystreams#Event';

/**
 * Flag
 * 
 * To flag something (e.g. flag as inappropriate, flag as spam, etc)
 *
 * http://www.w3.org/ns/activitystreams#Flag
 */
export const Flag: OwlClass = 'http://www.w3.org/ns/activitystreams#Flag';

/**
 * Follow
 * 
 * To Express Interest in Something
 *
 * http://www.w3.org/ns/activitystreams#Follow
 */
export const Follow: OwlClass = 'http://www.w3.org/ns/activitystreams#Follow';

/**
 * Group
 * 
 * A Group of any kind.
 *
 * http://www.w3.org/ns/activitystreams#Group
 */
export const Group: OwlClass = 'http://www.w3.org/ns/activitystreams#Group';

/**
 * Ignore
 * 
 * Actor is ignoring the Object
 *
 * http://www.w3.org/ns/activitystreams#Ignore
 */
export const Ignore: OwlClass = 'http://www.w3.org/ns/activitystreams#Ignore';

/**
 * Image
 * 
 * An Image file
 *
 * http://www.w3.org/ns/activitystreams#Image
 */
export const Image: OwlClass = 'http://www.w3.org/ns/activitystreams#Image';

/**
 * Invite
 * 
 * To invite someone or something to something
 *
 * http://www.w3.org/ns/activitystreams#Invite
 */
export const Invite: OwlClass = 'http://www.w3.org/ns/activitystreams#Invite';

/**
 * Join
 * 
 * To Join Something
 *
 * http://www.w3.org/ns/activitystreams#Join
 */
export const Join: OwlClass = 'http://www.w3.org/ns/activitystreams#Join';

/**
 * Leave
 * 
 * To Leave Something
 *
 * http://www.w3.org/ns/activitystreams#Leave
 */
export const Leave: OwlClass = 'http://www.w3.org/ns/activitystreams#Leave';

/**
 * Like
 * 
 * To Like Something
 *
 * http://www.w3.org/ns/activitystreams#Like
 */
export const Like: OwlClass = 'http://www.w3.org/ns/activitystreams#Like';

/**
 * View
 * 
 * The actor viewed the object
 *
 * http://www.w3.org/ns/activitystreams#View
 */
export const View: OwlClass = 'http://www.w3.org/ns/activitystreams#View';

/**
 * Listen
 * 
 * The actor listened to the object
 *
 * http://www.w3.org/ns/activitystreams#Listen
 */
export const Listen: OwlClass = 'http://www.w3.org/ns/activitystreams#Listen';

/**
 * Read
 * 
 * The actor read the object
 *
 * http://www.w3.org/ns/activitystreams#Read
 */
export const Read: OwlClass = 'http://www.w3.org/ns/activitystreams#Read';

/**
 * Move
 * 
 * The actor is moving the object. The target specifies where the object is moving to. The origin specifies where the object is moving from.
 *
 * http://www.w3.org/ns/activitystreams#Move
 */
export const Move: OwlClass = 'http://www.w3.org/ns/activitystreams#Move';

/**
 * Travel
 * 
 * The actor is traveling to the target. The origin specifies where the actor is traveling from.
 *
 * http://www.w3.org/ns/activitystreams#Travel
 */
export const Travel: OwlClass = 'http://www.w3.org/ns/activitystreams#Travel';

/**
 * Link
 * 
 * Represents a qualified reference to another resource. Patterned after the RFC5988 Web Linking Model
 *
 * http://www.w3.org/ns/activitystreams#Link
 */
export const Link: OwlClass = 'http://www.w3.org/ns/activitystreams#Link';

/**
 * Mention
 * 
 * A specialized Link that represents an @mention
 *
 * http://www.w3.org/ns/activitystreams#Mention
 */
export const Mention: OwlClass = 'http://www.w3.org/ns/activitystreams#Mention';

/**
 * Note
 * 
 * A Short note, typically less than a single paragraph. A "tweet" is an example, or a "status update"
 *
 * http://www.w3.org/ns/activitystreams#Note
 */
export const Note: OwlClass = 'http://www.w3.org/ns/activitystreams#Note';

/**
 * Object
 * 
 * null
 * Note that `Object` is a reserved Javascript keyword, and is therefore suffixed by `__workaround`.
 * For a list of reserved keywords, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords.
 *
 * http://www.w3.org/ns/activitystreams#Object
 */
export const Object__workaround: OwlClass = 'http://www.w3.org/ns/activitystreams#Object';

/**
 * Offer
 * 
 * To Offer something to someone or something
 *
 * http://www.w3.org/ns/activitystreams#Offer
 */
export const Offer: OwlClass = 'http://www.w3.org/ns/activitystreams#Offer';

/**
 * OrderedCollection
 * 
 * A variation of Collection in which items are strictly ordered
 *
 * http://www.w3.org/ns/activitystreams#OrderedCollection
 */
export const OrderedCollection: OwlClass = 'http://www.w3.org/ns/activitystreams#OrderedCollection';

/**
 * OrderedItems
 * 
 * A rdf:List variant for Objects and Links
 *
 * http://www.w3.org/ns/activitystreams#OrderedItems
 */
export const OrderedItems: OwlClass = 'http://www.w3.org/ns/activitystreams#OrderedItems';

/**
 * Page
 * 
 * A Web Page
 *
 * http://www.w3.org/ns/activitystreams#Page
 */
export const Page: OwlClass = 'http://www.w3.org/ns/activitystreams#Page';

/**
 * Person
 * 
 * A Person
 *
 * http://www.w3.org/ns/activitystreams#Person
 */
export const Person: OwlClass = 'http://www.w3.org/ns/activitystreams#Person';

/**
 * Organization
 * 
 * An Organization
 *
 * http://www.w3.org/ns/activitystreams#Organization
 */
export const Organization: OwlClass = 'http://www.w3.org/ns/activitystreams#Organization';

/**
 * Profile
 * 
 * A Profile Document
 *
 * http://www.w3.org/ns/activitystreams#Profile
 */
export const Profile: OwlClass = 'http://www.w3.org/ns/activitystreams#Profile';

/**
 * Place
 * 
 * A physical or logical location
 *
 * http://www.w3.org/ns/activitystreams#Place
 */
export const Place: OwlClass = 'http://www.w3.org/ns/activitystreams#Place';

/**
 * Question
 * 
 * A question of any sort.
 *
 * http://www.w3.org/ns/activitystreams#Question
 */
export const Question: OwlClass = 'http://www.w3.org/ns/activitystreams#Question';

/**
 * Reject
 * 
 * Actor rejects the Object
 *
 * http://www.w3.org/ns/activitystreams#Reject
 */
export const Reject: OwlClass = 'http://www.w3.org/ns/activitystreams#Reject';

/**
 * Remove
 * 
 * To Remove Something
 *
 * http://www.w3.org/ns/activitystreams#Remove
 */
export const Remove: OwlClass = 'http://www.w3.org/ns/activitystreams#Remove';

/**
 * Service
 * 
 * A service provided by some entity
 *
 * http://www.w3.org/ns/activitystreams#Service
 */
export const Service: OwlClass = 'http://www.w3.org/ns/activitystreams#Service';

/**
 * TentativeAccept
 * 
 * Actor tentatively accepts the Object
 *
 * http://www.w3.org/ns/activitystreams#TentativeAccept
 */
export const TentativeAccept: OwlClass = 'http://www.w3.org/ns/activitystreams#TentativeAccept';

/**
 * TentativeReject
 * 
 * Actor tentatively rejects the object
 *
 * http://www.w3.org/ns/activitystreams#TentativeReject
 */
export const TentativeReject: OwlClass = 'http://www.w3.org/ns/activitystreams#TentativeReject';

/**
 * Tombstone
 * 
 * A placeholder for a deleted object
 *
 * http://www.w3.org/ns/activitystreams#Tombstone
 */
export const Tombstone: OwlClass = 'http://www.w3.org/ns/activitystreams#Tombstone';

/**
 * Undo
 * 
 * To Undo Something. This would typically be used to indicate that a previous Activity has been undone.
 *
 * http://www.w3.org/ns/activitystreams#Undo
 */
export const Undo: OwlClass = 'http://www.w3.org/ns/activitystreams#Undo';

/**
 * Update
 * 
 * To Update/Modify Something
 *
 * http://www.w3.org/ns/activitystreams#Update
 */
export const Update: OwlClass = 'http://www.w3.org/ns/activitystreams#Update';

/**
 * Video
 * 
 * A Video document of any kind.
 *
 * http://www.w3.org/ns/activitystreams#Video
 */
export const Video: OwlClass = 'http://www.w3.org/ns/activitystreams#Video';

/**
 * actor
 * 
 * Subproperty of as:attributedTo that identifies the primary actor
 *
 * http://www.w3.org/ns/activitystreams#actor
 */
export const actor: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#actor';

/**
 * attributedTo
 * 
 * Identifies an entity to which an object is attributed
 *
 * http://www.w3.org/ns/activitystreams#attributedTo
 */
export const attributedTo: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#attributedTo';

/**
 * attachment
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#attachment
 */
export const attachment: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#attachment';

/**
 * attachments
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#attachments
 */
export const attachments: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#attachments';

/**
 * author
 * 
 * Identifies the author of an object. Deprecated. Use as:attributedTo instead
 *
 * http://www.w3.org/ns/activitystreams#author
 */
export const author: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#author';

/**
 * bcc
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#bcc
 */
export const bcc: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#bcc';

/**
 * bto
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#bto
 */
export const bto: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#bto';

/**
 * cc
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#cc
 */
export const cc: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#cc';

/**
 * context
 * 
 * Specifies the context within which an object exists or an activity was performed
 *
 * http://www.w3.org/ns/activitystreams#context
 */
export const context: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#context';

/**
 * current
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#current
 */
export const current: string = 'http://www.w3.org/ns/activitystreams#current';

/**
 * first
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#first
 */
export const first: string = 'http://www.w3.org/ns/activitystreams#first';

/**
 * generator
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#generator
 */
export const generator: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#generator';

/**
 * icon
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#icon
 */
export const icon: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#icon';

/**
 * image
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#image
 */
export const image: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#image';

/**
 * inReplyTo
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#inReplyTo
 */
export const inReplyTo: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#inReplyTo';

/**
 * items
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#items
 */
export const items: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#items';

/**
 * last
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#last
 */
export const last: string = 'http://www.w3.org/ns/activitystreams#last';

/**
 * location
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#location
 */
export const location: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#location';

/**
 * next
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#next
 */
export const next: string = 'http://www.w3.org/ns/activitystreams#next';

/**
 * object
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#object
 */
export const object: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#object';

/**
 * oneOf
 * 
 * Describes a possible exclusive answer or option for a question.
 *
 * http://www.w3.org/ns/activitystreams#oneOf
 */
export const oneOf: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#oneOf';

/**
 * oneOf
 * 
 * Describes a possible inclusive answer or option for a question.
 *
 * http://www.w3.org/ns/activitystreams#anyOf
 */
export const anyOf: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#anyOf';

/**
 * prev
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#prev
 */
export const prev: string = 'http://www.w3.org/ns/activitystreams#prev';

/**
 * preview
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#preview
 */
export const preview: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#preview';

/**
 * provider
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#provider
 */
export const provider: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#provider';

/**
 * replies
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#replies
 */
export const replies: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#replies';

/**
 * result
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#result
 */
export const result: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#result';

/**
 * audience
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#audience
 */
export const audience: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#audience';

/**
 * partOf
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#partOf
 */
export const partOf: string = 'http://www.w3.org/ns/activitystreams#partOf';

/**
 * tag
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#tag
 */
export const tag: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#tag';

/**
 * tags
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#tags
 */
export const tags: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#tags';

/**
 * target
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#target
 */
export const target: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#target';

/**
 * origin
 * 
 * For certain activities, specifies the entity from which the action is directed.
 *
 * http://www.w3.org/ns/activitystreams#origin
 */
export const origin: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#origin';

/**
 * instrument
 * 
 * Indentifies an object used (or to be used) to complete an activity
 *
 * http://www.w3.org/ns/activitystreams#instrument
 */
export const instrument: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#instrument';

/**
 * to
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#to
 */
export const to: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#to';

/**
 * url
 * 
 * Specifies a link to a specific representation of the Object
 *
 * http://www.w3.org/ns/activitystreams#url
 */
export const url: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#url';

/**
 * a
 * 
 * On a Relationship object, identifies the subject. e.g. when saying "John is connected to Sally", 'subject' refers to 'John'
 *
 * http://www.w3.org/ns/activitystreams#subject
 */
export const subject: string = 'http://www.w3.org/ns/activitystreams#subject';

/**
 * relationship
 * 
 * On a Relationship object, describes the type of relationship
 *
 * http://www.w3.org/ns/activitystreams#relationship
 */
export const relationship: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#relationship';

/**
 * describes
 * 
 * On a Profile object, describes the object described by the profile
 *
 * http://www.w3.org/ns/activitystreams#describes
 */
export const describes: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#describes';

/**
 * formerType
 * 
 * On a Tombstone object, describes the former type of the deleted object
 *
 * http://www.w3.org/ns/activitystreams#formerType
 */
export const formerType: OwlObjectProperty = 'http://www.w3.org/ns/activitystreams#formerType';

/**
 * accuracy
 * 
 * Specifies the accuracy around the point established by the longitude and latitude
 *
 * http://www.w3.org/ns/activitystreams#accuracy
 */
export const accuracy: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#accuracy';

/**
 * altitude
 * 
 * The altitude of a place
 *
 * http://www.w3.org/ns/activitystreams#altitude
 */
export const altitude: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#altitude';

/**
 * content
 * 
 * The content of the object.
 *
 * http://www.w3.org/ns/activitystreams#content
 */
export const content: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#content';

/**
 * name
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#name
 */
export const name: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#name';

/**
 * downstreamDuplicates
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#downstreamDuplicates
 */
export const downstreamDuplicates: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#downstreamDuplicates';

/**
 * duration
 * 
 * The duration of the object
 *
 * http://www.w3.org/ns/activitystreams#duration
 */
export const duration: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#duration';

/**
 * endTime
 * 
 * The ending time of the object
 *
 * http://www.w3.org/ns/activitystreams#endTime
 */
export const endTime: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#endTime';

/**
 * height
 * 
 * The display height expressed as device independent pixels
 *
 * http://www.w3.org/ns/activitystreams#height
 */
export const height: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#height';

/**
 * href
 * 
 * The target URI of the Link
 *
 * http://www.w3.org/ns/activitystreams#href
 */
export const href: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#href';

/**
 * hreflang
 * 
 * A hint about the language of the referenced resource
 *
 * http://www.w3.org/ns/activitystreams#hreflang
 */
export const hreflang: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#hreflang';

/**
 * id
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#id
 */
export const id: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#id';

/**
 * latitude
 * 
 * The latitude
 *
 * http://www.w3.org/ns/activitystreams#latitude
 */
export const latitude: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#latitude';

/**
 * longitude
 * 
 * The longitude
 *
 * http://www.w3.org/ns/activitystreams#longitude
 */
export const longitude: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#longitude';

/**
 * mediaType
 * 
 * The MIME Media Type
 *
 * http://www.w3.org/ns/activitystreams#mediaType
 */
export const mediaType: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#mediaType';

/**
 * objectType
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#objectType
 */
export const objectType: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#objectType';

/**
 * published
 * 
 * Specifies the date and time the object was published
 *
 * http://www.w3.org/ns/activitystreams#published
 */
export const published: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#published';

/**
 * radius
 * 
 * Specifies a radius around the point established by the longitude and latitude
 *
 * http://www.w3.org/ns/activitystreams#radius
 */
export const radius: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#radius';

/**
 * rating
 * 
 * A numeric rating (>= 0.0, <= 5.0) for the object
 *
 * http://www.w3.org/ns/activitystreams#rating
 */
export const rating: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#rating';

/**
 * rel
 * 
 * The RFC 5988 or HTML5 Link Relation associated with the Link
 *
 * http://www.w3.org/ns/activitystreams#rel
 */
export const rel: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#rel';

/**
 * startIndex
 * 
 * In a strictly ordered logical collection, specifies the index position of the first item in the items list
 *
 * http://www.w3.org/ns/activitystreams#startIndex
 */
export const startIndex: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#startIndex';

/**
 * startTime
 * 
 * The starting time of the object
 *
 * http://www.w3.org/ns/activitystreams#startTime
 */
export const startTime: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#startTime';

/**
 * summary
 * 
 * A short summary of the object
 *
 * http://www.w3.org/ns/activitystreams#summary
 */
export const summary: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#summary';

/**
 * totalItems
 * 
 * The total number of items in a logical collection
 *
 * http://www.w3.org/ns/activitystreams#totalItems
 */
export const totalItems: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#totalItems';

/**
 * units
 * 
 * Identifies the unit of measurement used by the radius, altitude and accuracy properties. The value can be expressed either as one of a set of predefined units or as a well-known common URI that identifies units.
 *
 * http://www.w3.org/ns/activitystreams#units
 */
export const units: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#units';

/**
 * updated
 * 
 * Specifies when the object was last updated
 *
 * http://www.w3.org/ns/activitystreams#updated
 */
export const updated: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#updated';

/**
 * upstreamDuplicates
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#upstreamDuplicates
 */
export const upstreamDuplicates: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#upstreamDuplicates';

/**
 * verb
 * 
 * 
 *
 * http://www.w3.org/ns/activitystreams#verb
 */
export const verb: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#verb';

/**
 * width
 * 
 * Specifies the preferred display width of the content, expressed in terms of device independent pixels.
 *
 * http://www.w3.org/ns/activitystreams#width
 */
export const width: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#width';

/**
 * deleted
 * 
 * Specifies the date and time the object was deleted
 *
 * http://www.w3.org/ns/activitystreams#deleted
 */
export const deleted: OwlDatatypeProperty = 'http://www.w3.org/ns/activitystreams#deleted';
